README.md

```diff
@@ -43,8 +43,7 @@ The library comes with a few tests like:
 
 ## Requirements
 
-GPUDirect RDMA requires NVIDIA Tesla or Quadro class GPUs based on Kepler,
-Pascal, Volta, or Turing, see [GPUDirect
+GPUDirect RDMA requires NVIDIA Data Center or Professional Graphics GPUs based on Kepler or newer generations, see [GPUDirect
 RDMA](http://developer.nvidia.com/gpudirect).  For more technical informations,
```

Comment:
could you please replace "more technical informations" with "more general information"?
That is because the RDMA whitepaper is way more technical than the marketing website.